tests/frame/test_common.py

```diff
@@ -355,6 +355,15 @@ def test_cross_join(df_raw: Any, expected: dict[str, list[Any]]) -> None:
         df.join(df, how="cross", left_on="a")  # type: ignore[arg-type]
 
 
+def test_cross_join_non_pandas() -> None:
```

Comment:
Locally I modified elsewhere as:

```diff
- if os.environ.get("CI") and get_modin() is not None:  # pragma: no cover
+ get_modin() is not None:  # pragma: no cover
    params.append(modin_constructor)
```

so that if you have modin installed it will cover that case

Reply:
nice! I think we can do that too anyway though? so we can get 100 coverage even without modin installed